make.py

```diff
@@ -65,7 +65,7 @@ def build():
   if not wasm:
     args += ' -s WASM=0 -s AGGRESSIVE_VARIABLE_ELIMINATION=1 -s ELIMINATE_DUPLICATE_FUNCTIONS=1 -s SINGLE_FILE=1 -s LEGACY_VM_SUPPORT=1'
   else:
-    args += ''' -s WASM=1 -s BINARYEN_IGNORE_IMPLICIT_TRAPS=1 -s BINARYEN_TRAP_MODE="clamp"'''
```

Comment:
What does this change do?

Reply:
see https://bugreports.qt.io/browse/QTBUG-76771 for make.py update

Reply:
more specifically, it allows building wasm with latest emscripten